snapcraft/tests/test_parser.py

```diff
@@ -408,3 +408,46 @@ def test_n_documents(
         main(['--debug', '--index', TEST_OUTPUT_PATH])
 
         self.assertEqual(2, _get_part_list_count())
+
+    @mock.patch('snapcraft.internal.parser._get_origin_data')
+    @mock.patch('snapcraft.internal.sources.get')
+    def test_maintaner_and_description_are_included(
```

Comment:
I split the test into two.  There's a lot of duplication but most of it is the mocked output.